e2e/internal/az/confcom.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+//go:build e2e
+
+package az
+
+import (
+	"os/exec"
+	"strings"
+)
+
+// KataPolicyGenVersion gets the version string of `az confcom katapolicygen`.
+func KataPolicyGenVersion() (string, error) {
+	out, err := exec.Command("az", "confcom", "katapolicygen", "--print-version").Output()
+	if err != nil {
+		return "", err
+	}
+	return strings.TrimSpace(string(out)), nil
+}
+
+// KataPolicyGen executes `az confcom katapolicygen --yaml <resourcePath>`.
+func KataPolicyGen(resourcePath string) error {
+	return exec.Command("az", "confcom", "katapolicygen", "--yaml", resourcePath).Run()
```

Comment:
Does this potentially swallow interesting command output?

Reply:
Not really, at most it swallows stderr which _could_ be interesting if the command fails. I've redirected the commands stderr output to the stderr output of the OS as a fix.